languages/civicrm.pot

```diff
@@ -1,192 +1,115 @@
-# Copyright (C) 2014 CiviCRM
+# Copyright (C) 2020 CiviCRM
 # This file is distributed under the same license as the CiviCRM package.
 msgid ""
 msgstr ""
-"Project-Id-Version: CiviCRM 4.6\n"
-"Report-Msgid-Bugs-To: http://wordpress.org/tag/civicrm\n"
-"POT-Creation-Date: 2014-11-11 09:48:56+00:00\n"
+"Project-Id-Version: CiviCRM 4.7\n"
```

Comment:
I'm not that familiar with what's required here. Just checking you are putting in 4.7 on the basis that the translations are good for 4.7+, rather than 5.x or 5.23 which might otherwise make more sense.

Reply:
@JoeMurray This file is generated by WP-CLI or the WordPress `makepot` bundle (the latter in this case) and the stated version is read from either [the entry in `readme.txt`](https://github.com/civicrm/civicrm-wordpress/blob/master/readme.txt#L6) or [the declared plugin version](https://github.com/civicrm/civicrm-wordpress/blob/master/civicrm.php#L5). 

It's not actually relevant to the plugin at this stage, since it's not hosted in the WordPress Plugin Directory. Refreshing this file just allows plugins like Loco Translate to pick up all (non CiviCRM Core) translatable strings in this plugin.